package.json

```diff
@@ -35,6 +35,11 @@
             "default": 5,
             "description": "How many seconds after last action counter freezes"
           },
+          "coding-timer.onlyWhenFocused": {
```

Comment:
How about renaming it to something like `coding-timer.freezeOnWindowBlur`? What do you think? Maybe that would ne as not confusing as we can be? I'm suggesting this as I saw that you changed around the way you describe it

Reply:
Agree, I wasn't so satisfied with the name